Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocAutoCompleteReferenceValueControlObject.cs

```diff
@@ -373,11 +376,7 @@ private IReadOnlyList<SearchServiceResultItem> ParseSearchServiceResponse ([NotN
       switch (state)
       {
         case AutoCompleteSearchService.Success:
-          var successData = (IReadOnlyCollection<object>)data;
-          return successData.Cast<IDictionary<string, object>>()
-              .Where(d => d != null) // empty JSON object (= no result) is converted to null
```

Comment:
Null-items get filtered out in the result. The new code doesn't do that because we can expect this to never happen?

Reply:
They do get filtered out in `ParseSearchServiceResultItems`

Reply:
I may be blind, so...how? (we can also check  this live)

Reply:
`OfType<>` filters `null` items.

Reply:
Ah, you switched from Cast to OfType.